tools/gen_nvfuser_version.py

```diff
@@ -11,14 +11,18 @@
 
 # note that this root currently is still part of pytorch.
 def get_sha() -> str:
+    import os
+
+    # assume the $NVFUSER_VERSION is in sha form
```

Comment:
Is it easy to check that? It seems quite easy to miss this and pass something like 0.2.3 instead.

Reply:
I don't think we have to assert it's in SHA format? Pytorch doesn't "assert" that either.

Reply:
Not quite straightforward to me, but I will give it a try.

Reply:
I would be surprised if someone has set NVFUSER_VERSION environment variable while trying to build nvfuser from source. 
Yeah Xiao is right, I am trying to do a version name override. By default, I would get version+gitUnknown when building nvfuser from an environment that git is not available.